Linting

djdanielsson opened the pull request:
ran ansible-lint --fix to see what it would suggest changing

Tompage1994 reviewed (approved):
Sure...

Not been through it all but it seems sensible enough

djdanielsson commented:
main ones I want to fix are the line length ones cuz I hate those

djdanielsson commented:
ok I updated this let me know what you think and if we should merge it or not

djdanielsson committed:
modified:   .ansible-lint
	modified:   .markdownlint.yml
	modified:   .pre-commit-config.yaml
	renamed:    .yamllint.yml -> .yamllint
	modified:   meta/runtime.yml
	modified:   playbooks/configure_aap.yml
	modified:   playbooks/get_stats.yml
	modified:   playbooks/rename_objects.yaml
	modified:   playbooks/set_organization.yaml
	modified:   roles/ansible_config/defaults/main.yml
	modified:   roles/ansible_config/meta/argument_specs.yml
	modified:   roles/ansible_config/meta/main.yml
	modified:   roles/ansible_config/tasks/main.yml
	modified:   roles/ansible_config/tests/test.yml
	modified:   roles/controller_ad_hoc_command/meta/argument_specs.yml
	modified:   roles/controller_ad_hoc_command/meta/main.yml
	modified:   roles/controller_ad_hoc_command/tasks/main.yml
	modified:   roles/controller_ad_hoc_command/tests/test.yml
	modified:   roles/controller_ad_hoc_command_cancel/meta/argument_specs.yml
	modified:   roles/controller_ad_hoc_command_cancel/meta/main.yml
	modified:   roles/controller_ad_hoc_command_cancel/tasks/main.yml
	modified:   roles/controller_ad_hoc_command_cancel/tests/test.yml
	modified:   roles/controller_applications/defaults/main.yml
	modified:   roles/controller_applications/meta/argument_specs.yml
	modified:   roles/controller_applications/meta/main.yml
	modified:   roles/controller_applications/tasks/main.yml
	modified:   roles/controller_applications/tests/test.yml
	modified:   roles/controller_bulk_host_create/defaults/main.yml
	modified:   roles/controller_bulk_host_create/meta/argument_specs.yml
	modified:   roles/controller_bulk_host_create/meta/main.yml
	modified:   roles/controller_bulk_host_create/tasks/main.yml
	modified:   roles/controller_bulk_job_launch/meta/argument_specs.yml
	modified:   roles/controller_bulk_job_launch/meta/main.yml
	modified:   roles/controller_bulk_job_launch/tasks/main.yml
	modified:   roles/controller_credential_input_sources/defaults/main.yml
	modified:   roles/controller_credential_input_sources/meta/argument_specs.yml
	modified:   roles/controller_credential_input_sources/meta/main.yml
	modified:   roles/controller_credential_input_sources/tasks/main.yml
	modified:   roles/controller_credential_input_sources/tests/test.yml
	modified:   roles/controller_credential_types/defaults/main.yml
	modified:   roles/controller_credential_types/meta/argument_specs.yml
	modified:   roles/controller_credential_types/meta/main.yml
	modified:   roles/controller_credential_types/tasks/main.yml
	modified:   roles/controller_credential_types/tests/test.yml
	modified:   roles/controller_credentials/defaults/main.yml
	modified:   roles/controller_credentials/meta/argument_specs.yml
	modified:   roles/controller_credentials/meta/main.yml
	modified:   roles/controller_credentials/tasks/main.yml
	modified:   roles/controller_credentials/tests/test.yml
	modified:   roles/controller_execution_environments/defaults/main.yml
	modified:   roles/controller_execution_environments/meta/argument_specs.yml
	modified:   roles/controller_execution_environments/meta/main.yml
	modified:   roles/controller_execution_environments/tasks/main.yml
	modified:   roles/controller_execution_environments/tests/test.yml
	modified:   roles/controller_host_groups/defaults/main.yml
	modified:   roles/controller_host_groups/meta/argument_specs.yml
	modified:   roles/controller_host_groups/meta/main.yml
	modified:   roles/controller_host_groups/tasks/main.yml
	modified:   roles/controller_host_groups/tests/test.yml
	modified:   roles/controller_hosts/defaults/main.yml
	modified:   roles/controller_hosts/meta/argument_specs.yml
	modified:   roles/controller_hosts/meta/main.yml
	modified:   roles/controller_hosts/tasks/main.yml
	modified:   roles/controller_hosts/tests/test.yml
	modified:   roles/controller_instance_groups/defaults/main.yml
	modified:   roles/controller_instance_groups/meta/argument_specs.yml
	modified:   roles/controller_instance_groups/meta/main.yml
	modified:   roles/controller_instance_groups/tasks/main.yml
	modified:   roles/controller_instance_groups/tests/test.yml
	modified:   roles/controller_instances/defaults/main.yml
	modified:   roles/controller_instances/meta/argument_specs.yml
	modified:   roles/controller_instances/meta/main.yml
	modified:   roles/controller_instances/tasks/main.yml
	modified:   roles/controller_instances/tests/test.yml
	modified:   roles/controller_inventories/defaults/main.yml
	modified:   roles/controller_inventories/meta/argument_specs.yml
	modified:   roles/controller_inventories/meta/main.yml
	modified:   roles/controller_inventories/tasks/main.yml
	modified:   roles/controller_inventories/tests/configs/inventories.yml
	modified:   roles/controller_inventories/tests/test.yml
	modified:   roles/controller_inventory_source_update/defaults/main.yml
	modified:   roles/controller_inventory_source_update/meta/argument_specs.yml
	modified:   roles/controller_inventory_source_update/meta/main.yml
	modified:   roles/controller_inventory_source_update/tasks/main.yml
	modified:   roles/controller_inventory_source_update/tests/test.yml
	modified:   roles/controller_inventory_sources/defaults/main.yml
	modified:   roles/controller_inventory_sources/meta/argument_specs.yml
	modified:   roles/controller_inventory_sources/meta/main.yml
	modified:   roles/controller_inventory_sources/tasks/main.yml
	modified:   roles/controller_inventory_sources/tests/test.yml
	modified:   roles/controller_job_launch/meta/argument_specs.yml
	modified:   roles/controller_job_launch/meta/main.yml
	modified:   roles/controller_job_launch/tasks/main.yml
	modified:   roles/controller_job_launch/tests/test.yml
	modified:   roles/controller_job_templates/defaults/main.yml
	modified:   roles/controller_job_templates/meta/argument_specs.yml
	modified:   roles/controller_job_templates/meta/main.yml
	modified:   roles/controller_job_templates/tasks/async.yml
	modified:   roles/controller_job_templates/tasks/main.yml
	modified:   roles/controller_job_templates/tests/test.yml
	modified:   roles/controller_jobs_cancel/meta/argument_specs.yml
	modified:   roles/controller_jobs_cancel/meta/main.yml
	modified:   roles/controller_jobs_cancel/tasks/main.yml
	modified:   roles/controller_jobs_cancel/tests/test.yml
	modified:   roles/controller_labels/defaults/main.yml
	modified:   roles/controller_labels/meta/argument_specs.yml
	modified:   roles/controller_labels/meta/main.yml
	modified:   roles/controller_labels/tasks/main.yml
	modified:   roles/controller_labels/tests/test.yml
	modified:   roles/controller_license/meta/argument_specs.yml
	modified:   roles/controller_license/meta/main.yml
	modified:   roles/controller_license/tasks/main.yml
	modified:   roles/controller_license/tasks/manifest.yml
	modified:   roles/controller_license/tasks/subscription.yml
	modified:   roles/controller_license/tests/config/license.yml
	modified:   roles/controller_license/tests/test.yml
	modified:   roles/controller_notification_templates/defaults/main.yml
	modified:   roles/controller_notification_templates/meta/argument_specs.yml
	modified:   roles/controller_notification_templates/meta/main.yml
	modified:   roles/controller_notification_templates/tasks/main.yml
	modified:   roles/controller_notification_templates/tests/configs/notifications.yml
	modified:   roles/controller_notification_templates/tests/test.yml
	modified:   roles/controller_organizations/defaults/main.yml
	modified:   roles/controller_organizations/meta/argument_specs.yml
	modified:   roles/controller_organizations/meta/main.yml
	modified:   roles/controller_organizations/tasks/main.yml
	modified:   roles/controller_organizations/tests/test.yml
	modified:   roles/controller_project_update/defaults/main.yml
	modified:   roles/controller_project_update/meta/argument_specs.yml
	modified:   roles/controller_project_update/meta/main.yml
	modified:   roles/controller_project_update/tasks/main.yml
	modified:   roles/controller_project_update/tests/test.yml
	modified:   roles/controller_projects/defaults/main.yml
	modified:   roles/controller_projects/meta/argument_specs.yml
	modified:   roles/controller_projects/meta/main.yml
	modified:   roles/controller_projects/tasks/main.yml
	modified:   roles/controller_projects/tests/test.yml
	modified:   roles/controller_roles/defaults/main.yml
	modified:   roles/controller_roles/meta/argument_specs.yml
	modified:   roles/controller_roles/meta/main.yml
	modified:   roles/controller_roles/tasks/main.yml
	modified:   roles/controller_roles/tests/test.yml
	modified:   roles/controller_schedules/defaults/main.yml
	modified:   roles/controller_schedules/meta/argument_specs.yml
	modified:   roles/controller_schedules/meta/main.yml
	modified:   roles/controller_schedules/tasks/main.yml
	modified:   roles/controller_schedules/tests/test.yml
	modified:   roles/controller_settings/defaults/main.yml
	modified:   roles/controller_settings/meta/argument_specs.yml
	modified:   roles/controller_settings/meta/main.yml
	modified:   roles/controller_settings/tasks/main.yml
	modified:   roles/controller_settings/tests/configs/settings.yml
	modified:   roles/controller_settings/tests/test.yml
	modified:   roles/controller_teams/defaults/main.yml
	modified:   roles/controller_teams/meta/argument_specs.yml
	modified:   roles/controller_teams/meta/main.yml
	modified:   roles/controller_teams/tasks/main.yml
	modified:   roles/controller_teams/tests/configs/teams.yml
	modified:   roles/controller_teams/tests/test.yml
	modified:   roles/controller_users/defaults/main.yml
	modified:   roles/controller_users/meta/argument_specs.yml
	modified:   roles/controller_users/meta/main.yml
	modified:   roles/controller_users/tasks/main.yml
	modified:   roles/controller_users/tests/test.yml
	modified:   roles/controller_workflow_job_templates/defaults/main.yml
	modified:   roles/controller_workflow_job_templates/meta/argument_specs.yml
	modified:   roles/controller_workflow_job_templates/meta/main.yml
	modified:   roles/controller_workflow_job_templates/tasks/add_workflows_schema.yml
	modified:   roles/controller_workflow_job_templates/tasks/main.yml
	modified:   roles/controller_workflow_job_templates/tests/configs/controller_workflows.yml
	modified:   roles/controller_workflow_job_templates/tests/test.yaml
	modified:   roles/controller_workflow_launch/meta/argument_specs.yml
	modified:   roles/controller_workflow_launch/meta/main.yml
	modified:   roles/controller_workflow_launch/tasks/main.yml
	modified:   roles/controller_workflow_launch/tests/test.yml
	modified:   roles/dispatch/meta/argument_specs.yml
	modified:   roles/dispatch/meta/main.yml
	modified:   roles/dispatch/tasks/main.yml
	modified:   roles/dispatch/tests/configs/credentials.yml
	modified:   roles/dispatch/tests/configs/inventories.yml
	modified:   roles/dispatch/tests/test.yml
	modified:   roles/eda_controller_tokens/defaults/main.yml
	modified:   roles/eda_controller_tokens/meta/argument_specs.yml
	modified:   roles/eda_controller_tokens/meta/main.yml
	modified:   roles/eda_controller_tokens/tasks/main.yml
	modified:   roles/eda_controller_tokens/tests/test.yml
	modified:   roles/eda_credential_types/defaults/main.yml
	modified:   roles/eda_credential_types/meta/argument_specs.yml
	modified:   roles/eda_credential_types/meta/main.yml
	modified:   roles/eda_credential_types/tasks/main.yml
	modified:   roles/eda_credential_types/tests/test.yml
	modified:   roles/eda_credentials/defaults/main.yml
	modified:   roles/eda_credentials/meta/argument_specs.yml
	modified:   roles/eda_credentials/meta/main.yml
	modified:   roles/eda_credentials/tasks/main.yml
	modified:   roles/eda_credentials/tests/test.yml
	modified:   roles/eda_credentials/tests/vars/credentials.yml
	modified:   roles/eda_decision_environments/defaults/main.yml
	modified:   roles/eda_decision_environments/meta/argument_specs.yml
	modified:   roles/eda_decision_environments/meta/main.yml
	modified:   roles/eda_decision_environments/tasks/main.yml
	modified:   roles/eda_decision_environments/tests/test.yml
	modified:   roles/eda_event_streams/defaults/main.yml
	modified:   roles/eda_event_streams/meta/argument_specs.yml
	modified:   roles/eda_event_streams/meta/main.yml
	modified:   roles/eda_event_streams/tasks/main.yml
	modified:   roles/eda_event_streams/tests/test.yml
	modified:   roles/eda_event_streams/tests/vars/event_streams.yml
	modified:   roles/eda_projects/defaults/main.yml
	modified:   roles/eda_projects/meta/argument_specs.yml
	modified:   roles/eda_projects/meta/main.yml
	modified:   roles/eda_projects/tasks/main.yml
	modified:   roles/eda_projects/tests/test.yml
	modified:   roles/eda_rulebook_activations/defaults/main.yml
	modified:   roles/eda_rulebook_activations/meta/argument_specs.yml
	modified:   roles/eda_rulebook_activations/meta/main.yml
	modified:   roles/eda_rulebook_activations/tasks/main.yml
	modified:   roles/eda_rulebook_activations/tests/test.yml
	modified:   roles/eda_users/defaults/main.yml
	modified:   roles/eda_users/meta/argument_specs.yml
	modified:   roles/eda_users/meta/main.yml
	modified:   roles/eda_users/tasks/main.yml
	modified:   roles/eda_users/tests/test.yml
	modified:   roles/gateway_applications/defaults/main.yml
	modified:   roles/gateway_applications/meta/argument_specs.yml
	modified:   roles/gateway_applications/tasks/main.yml
	modified:   roles/gateway_applications/tests/test.yml
	modified:   roles/gateway_authenticator_maps/defaults/main.yml
	modified:   roles/gateway_authenticator_maps/meta/argument_specs.yml
	modified:   roles/gateway_authenticator_maps/tasks/main.yml
	modified:   roles/gateway_authenticators/defaults/main.yml
	modified:   roles/gateway_authenticators/meta/argument_specs.yml
	modified:   roles/gateway_authenticators/tasks/main.yml
	modified:   roles/gateway_http_ports/defaults/main.yml
	modified:   roles/gateway_http_ports/meta/argument_specs.yml
	modified:   roles/gateway_http_ports/tasks/main.yml
	modified:   roles/gateway_organizations/defaults/main.yml
	modified:   roles/gateway_organizations/meta/argument_specs.yml
	modified:   roles/gateway_organizations/tasks/main.yml
	modified:   roles/gateway_role_user_assignments/defaults/main.yml
	modified:   roles/gateway_role_user_assignments/meta/argument_specs.yml
	modified:   roles/gateway_role_user_assignments/tasks/main.yml
	modified:   roles/gateway_routes/defaults/main.yml
	modified:   roles/gateway_routes/meta/argument_specs.yml
	modified:   roles/gateway_routes/tasks/main.yml
	modified:   roles/gateway_service_clusters/defaults/main.yml
	modified:   roles/gateway_service_clusters/meta/argument_specs.yml
	modified:   roles/gateway_service_clusters/tasks/main.yml
	modified:   roles/gateway_service_keys/defaults/main.yml
	modified:   roles/gateway_service_keys/meta/argument_specs.yml
	modified:   roles/gateway_service_keys/tasks/main.yml
	modified:   roles/gateway_service_nodes/defaults/main.yml
	modified:   roles/gateway_service_nodes/meta/argument_specs.yml
	modified:   roles/gateway_service_nodes/tasks/main.yml
	modified:   roles/gateway_services/defaults/main.yml
	modified:   roles/gateway_services/meta/argument_specs.yml
	modified:   roles/gateway_services/tasks/main.yml
	modified:   roles/gateway_settings/meta/argument_specs.yml
	modified:   roles/gateway_teams/defaults/main.yml
	modified:   roles/gateway_teams/meta/argument_specs.yml
	modified:   roles/gateway_teams/tasks/main.yml
	modified:   roles/gateway_users/defaults/main.yml
	modified:   roles/gateway_users/meta/argument_specs.yml
	modified:   roles/gateway_users/tasks/main.yml
	modified:   roles/global_vars/defaults/main.yml
	modified:   roles/global_vars/meta/main.yml
	modified:   roles/hub_collection/defaults/main.yml
	modified:   roles/hub_collection/meta/argument_specs.yml
	modified:   roles/hub_collection/meta/main.yml
	modified:   roles/hub_collection/tasks/main.yml
	modified:   roles/hub_collection/tests/test.yml
	modified:   roles/hub_collection_remote/defaults/main.yml
	modified:   roles/hub_collection_remote/meta/argument_specs.yml
	modified:   roles/hub_collection_remote/meta/main.yml
	modified:   roles/hub_collection_remote/tasks/main.yml
	modified:   roles/hub_collection_remote/tests/test.yml
	modified:   roles/hub_collection_repository/defaults/main.yml
	modified:   roles/hub_collection_repository/meta/argument_specs.yml
	modified:   roles/hub_collection_repository/meta/main.yml
	modified:   roles/hub_collection_repository/tasks/main.yml
	modified:   roles/hub_collection_repository/tests/test.yml
	modified:   roles/hub_collection_repository/tests/vars/repositories.yml
	modified:   roles/hub_collection_repository_sync/defaults/main.yml
	modified:   roles/hub_collection_repository_sync/meta/argument_specs.yml
	modified:   roles/hub_collection_repository_sync/meta/main.yml
	modified:   roles/hub_collection_repository_sync/tasks/main.yml
	modified:   roles/hub_collection_repository_sync/tests/test.yml
	modified:   roles/hub_collection_repository_sync/tests/vars/repositories.yml
	modified:   roles/hub_ee_image/defaults/main.yml
	modified:   roles/hub_ee_image/meta/argument_specs.yml
	modified:   roles/hub_ee_image/meta/main.yml
	modified:   roles/hub_ee_image/tasks/main.yml
	modified:   roles/hub_ee_image/tests/test.yml
	modified:   roles/hub_ee_registry/defaults/main.yml
	modified:   roles/hub_ee_registry/meta/argument_specs.yml
	modified:   roles/hub_ee_registry/meta/main.yml
	modified:   roles/hub_ee_registry/tasks/main.yml
	modified:   roles/hub_ee_registry/tests/test.yml
	modified:   roles/hub_ee_registry_index/defaults/main.yml
	modified:   roles/hub_ee_registry_index/meta/argument_specs.yml
	modified:   roles/hub_ee_registry_index/meta/main.yml
	modified:   roles/hub_ee_registry_index/tasks/main.yml
	modified:   roles/hub_ee_registry_index/tests/test.yml
	modified:   roles/hub_ee_registry_sync/defaults/main.yml
	modified:   roles/hub_ee_registry_sync/meta/argument_specs.yml
	modified:   roles/hub_ee_registry_sync/meta/main.yml
	modified:   roles/hub_ee_registry_sync/tasks/main.yml
	modified:   roles/hub_ee_registry_sync/tests/test.yml
	modified:   roles/hub_ee_repository/defaults/main.yml
	modified:   roles/hub_ee_repository/meta/argument_specs.yml
	modified:   roles/hub_ee_repository/meta/main.yml
	modified:   roles/hub_ee_repository/tasks/main.yml
	modified:   roles/hub_ee_repository/tests/test.yml
	modified:   roles/hub_ee_repository_sync/defaults/main.yml
	modified:   roles/hub_ee_repository_sync/meta/argument_specs.yml
	modified:   roles/hub_ee_repository_sync/meta/main.yml
	modified:   roles/hub_ee_repository_sync/tasks/main.yml
	modified:   roles/hub_ee_repository_sync/tests/test.yml
	modified:   roles/hub_group/defaults/main.yml
	modified:   roles/hub_group/meta/argument_specs.yml
	modified:   roles/hub_group/meta/main.yml
	modified:   roles/hub_group/tasks/main.yml
	modified:   roles/hub_group/tests/test.yml
	modified:   roles/hub_group_roles/defaults/main.yml
	modified:   roles/hub_group_roles/meta/argument_specs.yml
	modified:   roles/hub_group_roles/meta/main.yml
	modified:   roles/hub_group_roles/tasks/main.yml
	modified:   roles/hub_group_roles/tests/test.yml
	modified:   roles/hub_namespace/defaults/main.yml
	modified:   roles/hub_namespace/meta/argument_specs.yml
	modified:   roles/hub_namespace/meta/main.yml
	modified:   roles/hub_namespace/tasks/main.yml
	modified:   roles/hub_namespace/tests/test.yml
	modified:   roles/hub_namespace/tests/vars/namespaces.yml
	modified:   roles/hub_publish/defaults/main.yml
	modified:   roles/hub_publish/meta/argument_specs.yml
	modified:   roles/hub_publish/meta/main.yml
	modified:   roles/hub_publish/tasks/main.yml
	modified:   roles/hub_publish/tests/test.yml
	modified:   roles/hub_role/defaults/main.yml
	modified:   roles/hub_role/meta/argument_specs.yml
	modified:   roles/hub_role/meta/main.yml
	modified:   roles/hub_role/tasks/main.yml
	modified:   roles/hub_role/tests/test.yml
	modified:   roles/hub_user/defaults/main.yml
	modified:   roles/hub_user/meta/argument_specs.yml
	modified:   roles/hub_user/meta/main.yml
	modified:   roles/hub_user/tasks/main.yml
	modified:   roles/hub_user/tests/test.yml
	modified:   roles/meta_dependency_check/meta/main.yml
	modified:   roles/meta_dependency_check/tasks/main.yml
	modified:   roles/meta_dependency_check/tests/test.yml
	modified:   tests/configs/controller/credential_input_sources.yml
	modified:   tests/configs/controller/credentials.yml
	modified:   tests/configs/controller/differential_items.yml
	modified:   tests/configs/controller/execution_environments.yml
	modified:   tests/configs/controller/inventories.yml
	modified:   tests/configs/controller/notifications.yml
	modified:   tests/configs/controller/roles.yml
	modified:   tests/configs/controller/schedule.yml
	modified:   tests/configs/controller/settings.yml
	modified:   tests/configs/controller/settings_individuale.yml
	modified:   tests/configs/controller/teams.yml
	modified:   tests/configs/controller/templates.yml
	modified:   tests/configs/gateway/authenticator_maps.yml
	modified:   tests/configs/gateway/service_cluster.yml
	modified:   tests/configs/gateway/services.yml
	modified:   tests/configs/hub/ah_ansible_config.yml
	modified:   tests/configs/hub/ah_auth.yml
	modified:   tests/configs/hub/ah_collections.yml
	modified:   tests/configs/hub/ah_namespaces.yml
	modified:   tests/configs/hub/ah_repository.yml
	modified:   tests/configs_export_model/credentials_export.yml
	modified:   tests/configs_export_model/differential_items.yml
	modified:   tests/configs_export_model/inventory_export.yml
	modified:   tests/configs_export_model/inventory_sources_export.yml
	modified:   tests/configs_export_model/job_templates_export.yml
	modified:   tests/configs_export_model/notifications_templates_export.yml
	modified:   tests/configs_export_model/organizations_export.yml
	modified:   tests/configs_export_model/projects_export.yml
	modified:   tests/configs_export_model/teams_export.yml
	modified:   tests/configs_export_model/workflows.yml
	modified:   tests/configure_controller.yml
	modified:   tests/configure_controller_export_model.yml
	modified:   tests/configure_platform.yml
	modified:   tests/tasks/ad_hoc_cancel.yml
	modified:   tests/tasks/differential.yml

djdanielsson committed:
modified:   roles/controller_credential_input_sources/tasks/main.yml
	modified:   roles/controller_credential_types/tasks/main.yml
	modified:   roles/controller_credentials/tasks/main.yml
	modified:   roles/controller_execution_environments/tasks/main.yml
	modified:   roles/controller_host_groups/tasks/main.yml
	modified:   roles/controller_hosts/tasks/main.yml
	modified:   roles/controller_instance_groups/tasks/main.yml
	modified:   roles/controller_instances/tasks/main.yml
	modified:   roles/controller_inventories/tasks/main.yml
	modified:   roles/controller_inventory_source_update/tasks/main.yml
	modified:   roles/controller_inventory_sources/tasks/main.yml
	modified:   roles/controller_job_templates/tasks/main.yml
	modified:   roles/controller_labels/tasks/main.yml
	modified:   roles/controller_notification_templates/tasks/main.yml
	modified:   roles/controller_organizations/tasks/main.yml
	modified:   roles/controller_projects/tasks/main.yml
	modified:   roles/controller_roles/tasks/main.yml
	modified:   roles/controller_schedules/tasks/main.yml
	modified:   roles/controller_users/tasks/main.yml
	modified:   roles/controller_workflow_job_templates/tasks/add_workflows_schema.yml
	modified:   roles/controller_workflow_job_templates/tasks/main.yml
	modified:   roles/dispatch/defaults/main.yml
	modified:   roles/eda_controller_tokens/tasks/main.yml
	modified:   roles/eda_credential_types/tasks/main.yml
	modified:   roles/eda_credentials/tasks/main.yml
	modified:   roles/eda_event_streams/tasks/main.yml
	modified:   roles/gateway_organizations/tasks/main.yml
	modified:   roles/gateway_role_user_assignments/tasks/main.yml
	modified:   roles/gateway_routes/tasks/main.yml
	modified:   roles/gateway_service_keys/tasks/main.yml
	modified:   roles/gateway_service_nodes/tasks/main.yml
	modified:   roles/gateway_services/tasks/main.yml
	modified:   roles/gateway_teams/tasks/main.yml
	modified:   roles/gateway_users/tasks/main.yml
	modified:   roles/hub_collection/tasks/main.yml
	modified:   roles/hub_ee_registry/tasks/main.yml
	modified:   roles/hub_ee_registry_index/tasks/main.yml
	modified:   roles/hub_ee_registry_sync/tasks/main.yml
	modified:   roles/hub_ee_repository/tasks/main.yml
	modified:   roles/hub_ee_repository_sync/tasks/main.yml
	modified:   roles/hub_group_roles/tasks/main.yml
	modified:   roles/hub_namespace/tasks/main.yml
	modified:   tests/configure_controller.yml
	modified:   tests/configure_controller_export_model.yml

djdanielsson commented on playbooks/set_organization.yaml:
```diff
@@ -1,21 +1,21 @@
-        regexp: '^(.*)organization: (.*)$'
-        replace: '\1organization: {{ new_organization }}'
+        regexp: "^(.*)organization: (.*)$"
+        replace: "\\1organization: {{ new_organization }}"
```
I wonder if this will break something

djdanielsson committed:
modified:   playbooks/get_stats.yml
	modified:   roles/controller_ad_hoc_command/meta/argument_specs.yml
	modified:   roles/controller_ad_hoc_command_cancel/meta/argument_specs.yml
	modified:   roles/controller_applications/meta/argument_specs.yml
	modified:   roles/controller_bulk_host_create/meta/argument_specs.yml
	modified:   roles/controller_bulk_job_launch/meta/argument_specs.yml
	modified:   roles/controller_credential_input_sources/meta/argument_specs.yml
	modified:   roles/controller_credential_types/meta/argument_specs.yml
	modified:   roles/controller_credentials/meta/argument_specs.yml
	modified:   roles/controller_execution_environments/meta/argument_specs.yml
	modified:   roles/controller_host_groups/meta/argument_specs.yml
	modified:   roles/controller_hosts/meta/argument_specs.yml
	modified:   roles/controller_instance_groups/meta/argument_specs.yml
	modified:   roles/controller_instances/meta/argument_specs.yml
	modified:   roles/controller_inventories/meta/argument_specs.yml
	modified:   roles/controller_inventory_source_update/meta/argument_specs.yml
	modified:   roles/controller_inventory_sources/meta/argument_specs.yml
	modified:   roles/controller_job_launch/meta/argument_specs.yml
	modified:   roles/controller_job_templates/meta/argument_specs.yml
	modified:   roles/controller_jobs_cancel/meta/argument_specs.yml
	modified:   roles/controller_labels/meta/argument_specs.yml
	modified:   roles/controller_license/meta/argument_specs.yml
	modified:   roles/controller_notification_templates/meta/argument_specs.yml
	modified:   roles/controller_organizations/meta/argument_specs.yml
	modified:   roles/controller_project_update/meta/argument_specs.yml
	modified:   roles/controller_projects/meta/argument_specs.yml
	modified:   roles/controller_roles/meta/argument_specs.yml
	modified:   roles/controller_schedules/meta/argument_specs.yml
	modified:   roles/controller_settings/meta/argument_specs.yml
	modified:   roles/controller_teams/meta/argument_specs.yml
	modified:   roles/controller_users/meta/argument_specs.yml
	modified:   roles/controller_workflow_job_templates/meta/argument_specs.yml
	modified:   roles/controller_workflow_launch/meta/argument_specs.yml
	modified:   roles/dispatch/meta/argument_specs.yml
	modified:   roles/eda_controller_tokens/meta/argument_specs.yml
	modified:   roles/eda_credential_types/meta/argument_specs.yml
	modified:   roles/eda_credentials/meta/argument_specs.yml
	modified:   roles/eda_decision_environments/meta/argument_specs.yml
	modified:   roles/eda_event_streams/meta/argument_specs.yml
	modified:   roles/eda_projects/meta/argument_specs.yml
	modified:   roles/eda_rulebook_activations/meta/argument_specs.yml
	modified:   roles/eda_users/meta/argument_specs.yml
	modified:   roles/gateway_applications/meta/argument_specs.yml
	modified:   roles/gateway_authenticator_maps/meta/argument_specs.yml
	modified:   roles/gateway_authenticators/meta/argument_specs.yml
	modified:   roles/gateway_http_ports/meta/argument_specs.yml
	modified:   roles/gateway_organizations/meta/argument_specs.yml
	modified:   roles/gateway_role_user_assignments/meta/argument_specs.yml
	modified:   roles/gateway_routes/meta/argument_specs.yml
	modified:   roles/gateway_service_clusters/meta/argument_specs.yml
	modified:   roles/gateway_service_keys/meta/argument_specs.yml
	modified:   roles/gateway_service_nodes/meta/argument_specs.yml
	modified:   roles/gateway_services/meta/argument_specs.yml
	modified:   roles/gateway_settings/meta/argument_specs.yml
	modified:   roles/gateway_teams/meta/argument_specs.yml
	modified:   roles/gateway_users/meta/argument_specs.yml
	modified:   roles/hub_collection/meta/argument_specs.yml
	modified:   roles/hub_collection_remote/meta/argument_specs.yml
	modified:   roles/hub_collection_repository/meta/argument_specs.yml
	modified:   roles/hub_collection_repository_sync/meta/argument_specs.yml
	modified:   roles/hub_ee_image/meta/argument_specs.yml
	modified:   roles/hub_ee_registry/meta/argument_specs.yml
	modified:   roles/hub_ee_registry_index/meta/argument_specs.yml
	modified:   roles/hub_ee_registry_sync/meta/argument_specs.yml
	modified:   roles/hub_ee_repository/meta/argument_specs.yml
	modified:   roles/hub_ee_repository_sync/meta/argument_specs.yml
	modified:   roles/hub_group/meta/argument_specs.yml
	modified:   roles/hub_group_roles/meta/argument_specs.yml
	modified:   roles/hub_namespace/meta/argument_specs.yml
	modified:   roles/hub_publish/meta/argument_specs.yml
	modified:   roles/hub_role/meta/argument_specs.yml
	modified:   roles/hub_user/meta/argument_specs.yml

djdanielsson committed:
modified:   roles/controller_credential_types/tasks/main.yml
	modified:   roles/controller_execution_environments/tasks/main.yml
	modified:   roles/controller_host_groups/tasks/main.yml
	modified:   roles/controller_hosts/tasks/main.yml
	modified:   roles/controller_instance_groups/tasks/main.yml
	modified:   roles/controller_instances/tasks/main.yml
	modified:   roles/controller_inventories/tasks/main.yml
	modified:   roles/controller_inventory_sources/tasks/main.yml
	modified:   roles/controller_job_templates/tasks/async.yml
	modified:   roles/controller_job_templates/tasks/main.yml
	modified:   roles/controller_license/tasks/main.yml
	modified:   roles/controller_license/tasks/manifest.yml
	modified:   roles/controller_notification_templates/tasks/main.yml
	modified:   roles/controller_organizations/tasks/main.yml
	modified:   roles/controller_projects/tasks/main.yml
	modified:   roles/controller_roles/tasks/main.yml
	modified:   roles/controller_schedules/tasks/main.yml
	modified:   roles/controller_users/tasks/main.yml
	modified:   roles/controller_workflow_job_templates/tasks/add_workflows_schema.yml
	modified:   roles/controller_workflow_job_templates/tasks/main.yml

djdanielsson committed:
modified:   CONVERSION_GUIDE.md
	modified:   EXPORT_README.md
	modified:   roles/ansible_config/README.md
	modified:   roles/controller_ad_hoc_command/README.md
	modified:   roles/controller_ad_hoc_command_cancel/README.md
	modified:   roles/controller_applications/README.md
	modified:   roles/controller_applications/meta/argument_specs.yml
	modified:   roles/controller_bulk_host_create/README.md
	modified:   roles/controller_bulk_job_launch/README.md
	modified:   roles/controller_credential_input_sources/README.md
	modified:   roles/controller_credential_types/README.md
	modified:   roles/controller_credentials/README.md
	modified:   roles/controller_credentials/meta/argument_specs.yml
	modified:   roles/controller_execution_environments/README.md
	modified:   roles/controller_host_groups/README.md
	modified:   roles/controller_hosts/README.md
	modified:   roles/controller_instance_groups/README.md
	modified:   roles/controller_instances/README.md
	modified:   roles/controller_inventories/README.md
	modified:   roles/controller_inventory_source_update/README.md
	modified:   roles/controller_inventory_sources/README.md
	modified:   roles/controller_job_launch/README.md
	modified:   roles/controller_job_launch/meta/argument_specs.yml
	modified:   roles/controller_job_templates/README.md
	modified:   roles/controller_jobs_cancel/README.md
	modified:   roles/controller_labels/README.md
	modified:   roles/controller_license/README.md
	modified:   roles/controller_notification_templates/README.md
	modified:   roles/controller_organizations/README.md
	modified:   roles/controller_project_update/README.md
	modified:   roles/controller_projects/README.md
	modified:   roles/controller_projects/meta/argument_specs.yml
	modified:   roles/controller_roles/README.md
	modified:   roles/controller_schedules/README.md
	modified:   roles/controller_settings/README.md
	modified:   roles/controller_teams/README.md
	modified:   roles/controller_users/README.md
	modified:   roles/controller_users/meta/argument_specs.yml
	modified:   roles/controller_workflow_job_templates/README.md
	modified:   roles/controller_workflow_launch/README.md
	modified:   roles/controller_workflow_launch/meta/argument_specs.yml
	modified:   roles/eda_controller_tokens/README.md
	modified:   roles/eda_credential_types/README.md
	modified:   roles/eda_credentials/README.md
	modified:   roles/eda_decision_environments/README.md
	modified:   roles/eda_event_streams/README.md
	modified:   roles/eda_projects/README.md
	modified:   roles/eda_rulebook_activations/README.md
	modified:   roles/eda_users/README.md
	modified:   roles/gateway_applications/README.md
	modified:   roles/gateway_applications/meta/argument_specs.yml
	modified:   roles/gateway_authenticator_maps/README.md
	modified:   roles/gateway_authenticators/README.md
	modified:   roles/gateway_http_ports/README.md
	modified:   roles/gateway_http_ports/meta/argument_specs.yml
	modified:   roles/gateway_organizations/README.md
	modified:   roles/gateway_role_user_assignments/README.md
	modified:   roles/gateway_routes/README.md
	modified:   roles/gateway_service_clusters/README.md
	modified:   roles/gateway_service_keys/README.md
	modified:   roles/gateway_service_nodes/README.md
	modified:   roles/gateway_services/README.md
	modified:   roles/gateway_settings/README.md
	modified:   roles/gateway_teams/README.md
	modified:   roles/gateway_users/README.md
	modified:   roles/gateway_users/meta/argument_specs.yml
	modified:   roles/hub_collection/README.md
	modified:   roles/hub_collection_remote/README.md
	modified:   roles/hub_collection_repository/README.md
	modified:   roles/hub_collection_repository_sync/README.md
	modified:   roles/hub_ee_image/README.md
	modified:   roles/hub_ee_registry/README.md
	modified:   roles/hub_ee_registry_index/README.md
	modified:   roles/hub_ee_registry_sync/README.md
	modified:   roles/hub_ee_repository/README.md
	modified:   roles/hub_ee_repository_sync/README.md
	modified:   roles/hub_group/README.md
	modified:   roles/hub_group_roles/README.md
	modified:   roles/hub_namespace/README.md
	modified:   roles/hub_publish/README.md
	modified:   roles/hub_role/README.md
	modified:   roles/hub_user/README.md
	modified:   tests/configs/controller/settings.yml
	modified:   tests/configs/controller/settings_individuale.yml
	modified:   tests/templated_role_example/README.md
